.circleci/config.yml

```diff
@@ -1,23 +0,0 @@
-version: 2.0
-jobs:
-  build:
-    docker:
-      - image: circleci/golang:1.12
-    working_directory: /go/src/github.com/sparrc/go-ping
-    steps:
-      - checkout
-      - run:
-          name: Build and run some basic tests
-          command: |
-            go get ./...
-            go vet ./...
-            go test ./... -cover -race
-            go build -race -o ping_linux ./cmd/ping/ping.go
-            sudo ./ping_linux --privileged -c 2 www.google.com
-            sudo ./ping_linux --privileged -c 3 -i 200ms www.google.com
-            sudo ./ping_linux --privileged -c 10 -i 100ms -t 1s www.google.com
```

Comment:
re-requesting review because github actions doesn't appear to support this part of the circle ci job, seems like it's because outside internet connectivity is limited within workflow jobs.

Reply:
Github actions or CircleCI? I'm confused.

Reply:
Running the ping command was part of the circleci job. Migrating to actions requires us to remove running the ping command.

Reply:
I wonder if external ICMP connectivity can be enabled if we get in touch with GitHub support?

Reply:
I'm wondering why switch from CircleCI to GH actions?

Reply:
the only reason was to use github's builtin support, but it may not be worth it if we're missing features

Reply:
okay Ive backed out the github actions changes, perhaps we can revisit at a later time